appveyor.yml

```diff
@@ -28,3 +28,19 @@ after_build:
 
 build_script:
 - cmd: build-vs2017.cmd
+
+artifacts:
+- path: build\*.nupkg
+  name: core
+
+deploy:
+- provider: NuGet
+  api_key:
+    secure: Please insert key here
+  skip_symbols: false
+  artifact: core 
+  on:
+      appveyor_repo_tag: true
```

Comment:
I'm trying to understand how the version is set, the docs (https://www.appveyor.com/docs/branches/) make no reference to version, only `APPVEYOR_REPO_TAG` and `APPVEYOR_REPO_TAG_NAME`.